src/core/ext/xds/xds_bootstrap_grpc.cc

```diff
@@ -227,8 +250,10 @@ absl::StatusOr<std::unique_ptr<GrpcXdsBootstrap>> GrpcXdsBootstrap::Create(
     absl::string_view json_string) {
   auto json = JsonParse(json_string);
   if (!json.ok()) {
-    return absl::InvalidArgumentError(absl::StrCat(
-        "Failed to parse bootstrap JSON string: ", json.status().ToString()));
+    return absl::InvalidArgumentError(
+        absl::StrCat("Failed to parse bootstrap "
+                     "JSON string: ",
```

Comment:
Why split this string across lines?  Seems like it was more readable how it was before.